include/glow/Runtime/RuntimeTypes.h

```diff
@@ -143,6 +143,18 @@ struct DeviceConfig {
       : backendKind(kind), name(name), parameters(parameters) {}
 
   bool hasName() const { return name != ""; }
+
+  void setDeviceSize(uint64_t mem_size) {
+    parameters["device_size"] = std::to_string(mem_size);
+  }
+
+  uint64_t getDeviceSize() {
+    uint64_t ret = 0;
+    if (parameters.find("device_size") != parameters.end()) {
+      ret = std::stoull(parameters["device_size"]);
```

Comment:
Instead of performing another lookup if the key is in the map, save the iterator returned by `find` and use that in the body of the if statement:

```
auto it = parameters.find("device_size");
if (it != parameters.end()) {
      ret = std::stoull(it->second);
}
```